lib/msal-angular/README.md

```diff
@@ -1,7 +1,7 @@
 # Microsoft Authentication Library for Angular
 
 
-Note: `msal-angular@2` is under development. **We do not recommend using this version in a production environment yet**.
+Note: MSAL Angular v2 is under development. **We do not recommend using this version in a production environment yet**.
```

Comment:
For public preview we can remove the warning to note use in prod.

Reply:
Noted. Done